src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -2353,6 +2357,15 @@ public function get_item_schema() {
 				'context'     => array( 'edit' ),
 				'readonly'    => true,
 			);
+
+			$schema['properties']['class_list'] = array(
```

Comment:
We should add a `readonly` keyword.

Reply:
Good catch. Fixed in e4c1641c551520014364fa7f95442eff277b27a8
Thanks.